src/SignalRServiceExtension/Bindings/SignalRAsyncCollectorBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+    internal class SignalRAsyncCollectorBuilder<T> : IAsyncConverter<SignalRAttribute, IAsyncCollector<T>>
```

Comment:
>SignalRAsyncCollectorBuilder [](start = 19, length = 28)

revert?

Reply:
Why?

Reply:
This is a new class. The original class is without `Async`.